packages/grid/src/Grid.tsx

```diff
@@ -850,6 +832,37 @@ class Grid extends PureComponent<GridProps, GridState> {
     }
   }
 
+  checkMetricsUpdates() {
```

Comment:
Can you add a comment for why you would call this method over `updateMetrics`? Seems this is actually just scrolling sticky tables. If that's the case, update the method name to be more clear. `scrollIfSticky` maybe?